modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java

```diff
@@ -88,32 +93,22 @@ public class ServiceOrderEntity implements Serializable {
     @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss XXX")
     private OffsetDateTime completedTime;
 
-    @Column(name = "PREVIOUS_DEPLOY_REQUEST", columnDefinition = "json")
-    @Type(value = JsonType.class)
-    @Convert(converter = ObjectJsonConverter.class)
-    private DeployRequest previousDeployRequest;
-
-    @Column(name = "PREVIOUS_DEPLOYED_RESOURCES", columnDefinition = "json")
-    @Type(value = JsonType.class)
-    @Convert(converter = ObjectJsonConverter.class)
-    private List<DeployResource> previousDeployedResources;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "HANDLER", nullable = false)
```

Comment:
we do not need this as well here. 
Moreover, one order can have multiple handlers.